website/docs/artist_hosts_tvpaint.md

```diff
@@ -97,27 +85,45 @@ Render Layer bakes all the animation layers of one particular color group togeth
 
 </div>
 </div>
+<br/>
+
+**How to mark TVPaint layer to a group**
+
+In the bottom left corner of your timeline, you will note a **Color group** button.
 
+![colorgroups](assets/tvp_color_groups.png)
 
+It allows you to choose a group by checking one of the colors of the color list.
 
+![colorgroups](assets/tvp_color_groups2.png)
+
+The timeline's animation layer can be marked by the color you pick from your Color group. Layers in the timeline with the same color are gathered into a group represents one render layer.
+
+![timeline](assets/tvp_timeline_color.png)
 
 
 ### Render Pass
 
-Render Passes are smaller individual elements of a Render Layer. A `character` render layer might
+Render Passes are smaller individual elements of a [Render Layer](artist_hosts_tvpaint.md#render-layer). A `character` render layer might
 consist of multiple render passes such as `Line`, `Color` and `Shadow`.
 
+Render Passes are specific because they have to belong to a particular Render Layer. You have to select to which Render Layer the pass belongs. Try to refresh if you don't see demanded Render Layer in the options.
```

Comment:
```suggestion
Render Passes are specific because they have to belong to a particular Render Layer. You have to select to which Render Layer the pass belongs. Try to refresh if you don't see a specific Render Layer in the options.
```